### DISCUSSION TITLE
Some or all destination ports are invalid: [![80,443]"}

### DISCUSSION THREAD

**Discussion by muskan399** [google/gonids#192] [General]
While parsing a rule with `..  any -> any ![80,443] ..`, getting some or all destination ports are invalid: [![80,443]"}, as error

**Comment by duanehoward** [comment]
This is likely to be related to https://github.com/google/gonids/issues/156. The network parsing code doesn't handle some less common patterns well and modifying the lexer and parser to correctly handle these has been more difficult than anticipated. Along with that I've not had spare time to allocate to this project for quite some time, but I try to review pull requests in a timely fashion.
